launch

```diff
@@ -51,9 +52,12 @@ def DevelRun(sim, dbg, uarts):
         session.select_window(dbg.name if dbg else '/dev/cons')
         session.attach_session()
     finally:
-        session.kill_session()
         server.kill_server()
 
+        # Give QEMU a chance to exit gracefully
+        time.sleep(0.2)
```

Comment:
Could we call `sim.stop()` here instead?

Reply:
`sim.stop()` kills the process created by the `subprocess` in the `sim.run()` procedure.
The problem is that QEMU is managed by tmux, so we can't stop it using `subprocess`.